marklogic-data-hub/src/main/java/com/marklogic/hub/impl/MarkLogicVersion.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2020 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.hub.impl;
```

Comment:
Sorry, one more tweak here - in expectation of this class going into the API project, let's put it in "com.marklogic.hub". The feature provided by this class is generic to all of DHF, so I think that root package is justified here.

Reply:
Moved under com.marklogic.hub